it/helpers/AuthStubs.scala

```diff
@@ -73,7 +73,7 @@ trait AuthStubs {
       Json.obj("allEnrolments" -> enrolments)
   }
 
-  def authorised(response: JsObject = successfulAuthResponse(Some(Individual), Some(ConfidenceLevel.L200),mtditEnrolment,ninoEnrolment)): StubMapping = {
+  def authorised(response: JsObject = successfulAuthResponse(Some(Individual), Some(ConfidenceLevel.L250),mtditEnrolment,ninoEnrolment)): StubMapping = {
```

Comment:
I would believe you need to extract this from config?

Reply:
Done